### ISSUE TITLE
Use a minimum of 25% completness for translations

### ISSUE THREAD

**Issue by sikmir** [OpenOrienteering/mapper#794]
Now we have some languages in Settings/Language list, which has <1% translation. It makes user confused a bit. It looks reasonable to not ship empty translations (and even <25% translations as not ready for release).
For example, in [Transifex](https://www.transifex.com) it can be easy done by _minimum_perc_ option:
```
$ cat .tx/config
[main]
host = https://www.transifex.com

[QGIS.qgis-application]
file_filter = i18n/qgis_<lang>.ts
minimum_perc = 35
...
```

> **minimum_perc** - A minimum completion percentage that translations must meet in order to be downloaded; defining this param here sets the minimum_perc param for all project resources unless overridden in the resource config or when using the --minimum_perc=VALUE param when issuing the tx pull command (optional param).

**Comment by dg0yt** [comment]
I am aware of the issue. I did not take any measure after being accepted for hosted weblate in order to encourage contributors, but I will prune the list of released languages for 0.7.x.

In addition, I recently changed our weblate settings to block the direct creation of new translations there.

**Comment by dg0yt** [comment]
By the way, is there a Qt translation for Esperanto?

**Comment by sikmir** [comment]
Not yet.

**Comment by dg0yt** [comment]
At the moment I add a collection of ~300 Qt strings to these translations (standard dialog texts, error messages, keys etc.). I will do this for Esperanto soon.

**Comment by dg0yt** [comment]
Effectively resolved in c571fd3. But there is no automatism.
